resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js

```
@@ -74,10 +73,6 @@ const ScholarshipInfoBlock = ({
   );
   const actionType = get(actionItem, 'actionLabel', '');
 
-  if (error) {
-    return <ErrorBlock error={error} />;
-  }
-
```

Ok, now I'm just being annoying, but I wonder if it might still be worth logging and reporting this error like we do from the `ErrorBlock`
https://github.com/DoSomething/phoenix-next/blob/7837ca6ad174868a3ad9d75b9ae3276440166dd3/resources/assets/components/blocks/ErrorBlock/ErrorBlock.js#L13-L16
(maybe without the `[ErrorBlock]` bit in the console log)
This way the request can silently fail without erroring out the info block, but we'd still know that the error took place 🤔

---

😂 i like this idea! I'm going to add this in before merging